koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -709,6 +709,64 @@ def populate_monthly_cost(  # noqa: C901
                         cost_category_map,
                     )
 
+    def populate_monthly_tag_cost_sql(  # noqa: C901
+        self, cost_type, rate_type, tag_key, case_dict, start_date, end_date, distribution, provider_uuid
+    ):
+        """
+        Update or insert daily summary line item for node cost.
+        It checks to see if a line item exists for each node
+        that contains the tag key:value pair,
+        if it does then the price is added to the monthly cost.
+        """
+        table_name = self._table_map["line_item_daily_summary"]
+        report_period = self.report_periods_for_provider_uuid(provider_uuid, start_date)
+        with schema_context(self.schema):
+            report_period_id = report_period.id
+
+        cpu_case, memory_case, volume_case = case_dict.get("allocated")
+        labels = case_dict.get("labels")
+
+        if cost_type == "Node":
+            summary_sql = pkgutil.get_data("masu.database", "sql/openshift/cost_model/monthly_cost_node_by_tag.sql")
```

Comment:
It's a good convention to prefix vars containing bytes with `b_` so it's clear which vars are bytes and which are text. Not something to do in this PR, but something to think about.